railgun/internal/proxy/clientgo_cached_proxy_resolver.go

```diff
@@ -142,6 +142,7 @@ type clientgoCachedProxyResolver struct {
 
 	// locks
 	internalCacheLock *sync.RWMutex
+	backendNeedsSync  bool
```

Comment:
can this be a local variable in `startCacheServer`?

Reply:
For clarification: I am expecting the `backendNeedsSync` declaration to be removed from the definition of `struct clientgoCachedProxyResolver` and to be placed as a local variable in `startCacheServer` instead.

Reply:
Originally I had this in the struct so that I could manipulate it in tests, but the tests ended up not really needing it.

This is changed now :+1:

Reply:
f6d49621fd4229a108b11b56f644b811dd568764